docs/quickstart/create-and-publish-a-package-using-visual-studio-net-framework.md

```diff
@@ -1,162 +1,176 @@
-        <iconUrl>http://ICON_URL_HERE_OR_DELETE_THIS_LINE</iconUrl>
+        <projectUrl>http://ENTER_PROJECT_URL_HERE_OR_DELETE_THIS_LINE</projectUrl>
+        <iconUrl>http://ENTER_ICON_URL_HERE_OR_DELETE_THIS_LINE</iconUrl>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>Package description</description>
         <releaseNotes>Summary of changes made in this release of the package.</releaseNotes>
-        <copyright>Copyright 2019</copyright>
+        <copyright>Copyright 2022</copyright>
         <tags>Tag1 Tag2</tags>
       </metadata>
     </package>
     ```
 
 ## Edit the manifest
 
-1. NuGet produces an error if you try to create a package with default values in your `.nuspec` file, so you must edit the following fields before proceeding. See [.nuspec file reference - optional metadata elements](../reference/nuspec.md#optional-metadata-elements) for a description of how these are used.
+1. Edit the following properties before proceeding. Otherwise, if you try to create a NuGet package with the default values in your `.nuspec` file, an error occurs. For information about these properties, see [Optional metadata elements](../reference/nuspec.md#optional-metadata-elements):
 
     - licenseUrl
     - projectUrl
     - iconUrl
     - releaseNotes
     - tags
 
-1. For packages built for public consumption, pay special attention to the **Tags** property, as tags help others find your package on sources like nuget.org and understand what it does.
+1. For packages built for public consumption, pay special attention to the **Tags** property, as tags help others find your package and understand what it does.
 
-1. You can also add any other elements to the manifest at this time, as described on [.nuspec file reference](../reference/nuspec.md).
+1. You can also add any other elements to the manifest at this time, as described in [.nuspec file reference](../reference/nuspec.md).
 
 1. Save the file before proceeding.
 
 ## Run the pack command
 
-1. From a command prompt in the folder containing your `.nuspec` file, run the command `nuget pack`.
+1. With your project open in **Solution Explorer**, open a command prompt by selecting **Tools** > **Command Line** >  **Developer Command Prompt**.
+
+   The command prompt opens in your project directory.
 
-1. NuGet generates a `.nupkg` file in the form of *identifier-version.nupkg*, which you'll find in the current folder.
+1. Run the following command: `nuget pack`.
+
+   NuGet generates a *.nupkg* file in the form of *identifier-version.nupkg* in the current folder.
 
 ## Publish the package
 
-Once you have a `.nupkg` file, you publish it to nuget.org using `nuget.exe` with an API key acquired from nuget.org. For nuget.org you must use `nuget.exe` 4.1.0 or higher.
+After you've created a *.nupkg* file, publish it to nuget.org by using the NuGet CLI with an API key acquired from nuget.org. For nuget.org, you must use `nuget.exe` 4.1.0 or higher.
```

Comment:
the nuget.org FAQ has an entry about test packages: https://docs.microsoft.com/en-us/nuget/nuget-org/nuget-org-faq#is-it-recommended-to-upload-my-test-packages-to-nuget-org

it suggests using int.nugettest.org, rather than nuget.org.  Perhaps the quickstart should follow the same recommendation?